pkg/config/constants/constants.go

```diff
@@ -175,4 +175,5 @@ const (
 	AmbientRedirectionEnabled = "enabled"
 	// AmbientRedirectionDisabled is an opt-out, configured by user.
 	AmbientRedirectionDisabled = "disabled"
+	AmbientRedirectionNone     = "null"
```

Comment:
This is confusing imo, it makes it seem like `"null"` is a valid value. But its not -- `null` is just how you remove a value in the path.